lib/mix/tasks/phx.gen.auth.ex

```diff
@@ -483,19 +483,14 @@ defmodule Mix.Tasks.Phx.Gen.Auth do
         {:error, _} -> "use Mix.Config\n"
       end
 
-    with :error <- split_with_self(contents, "use Mix.Config\n"),
```

Comment:
You can actually pass a list to `:binary.split`, so perhaps this is better written by passing as argument `["use Mix.Config\r\n", "use Mix.Config\n"]`. Thoughts?

Reply:
Another option is to change `File.read(file)` above and have it return `{:ok, bin} -> String.replace(bin, "\r\n", "\n")`.

Reply:
The latter would change all the newlines in the file, which seems kinda rude. 
However the former might create a mix of newline styles which is also a problem with the original pull requests. I guess ideally we would honor the newline style of the file (if the file itself is consistent?)
However at this point it might be overthinking 😄